crates/locutus-node/src/node/tracer.rs

```diff
@@ -0,0 +1,20 @@
+use opentelemetry::sdk::trace::Tracer;
+use opentelemetry::{global, trace::{TraceError, TracerProvider}};
+use opentelemetry::sdk::propagation::TraceContextPropagator;
+use tracing_opentelemetry::OpenTelemetryLayer;
+use tracing_subscriber::layer::Layered;
+use tracing_subscriber::prelude::*;
+use tracing_subscriber::Registry;
+
+pub fn init_tracer(service_name: String) -> Result<(), TraceError> {
+    let tracer = opentelemetry_jaeger::new_pipeline()
+        .with_service_name(service_name)
+        //.with_agent_endpoint("127.0.0.1:6831")
```

Comment:
remember to pass this via config (we can add a new environment variable for now)